test/unit/data/test_metdata_limit.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+import galaxy.datatypes.registry as registry
+import galaxy.model.mapping as mapping
+from galaxy.model import (
+    custom_types,
+    HistoryDatasetAssociation,
+    set_datatypes_registry,
+)
+
+METADATA_LIMIT = 500
+
+
+@pytest.fixture(scope="module")
+def datatypes_registry():
+    r = registry.Registry()
+    r.load_datatypes()
+    set_datatypes_registry(r)
+
+
+@pytest.fixture
+def sa_session(datatypes_registry):
+    custom_types.MAX_METADATA_VALUE_SIZE = METADATA_LIMIT
```

Comment:
I think this can be moved outside of the function to line 12?

Reply:
Then it runs on import, I don't think that's an improvement ?

Reply:
I don't think these are supposed to be imported, but we can leave it as it is just in case.